docs/application/tizen-studio/release-notes/2-5-release-notes.md

```diff
@@ -0,0 +1,82 @@
+# Tizen Studio 2.5 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   SDB
+	-  wait-for-device command added, when this command is issued sdb waits indefinitely for the target to come to 'device' state.
+-   RT IDE Certificate Manager
+	-   Integrated Certificate Manager into RT IDE
+-   Baseline SDK Size Optimizations
+	-   Optimized Baseline SDK size from 480MB to 320MB
+
+### Fixed Bugs
+-   Certificate Manager
+	-   Added validation of .pem & .der files while importing as a new IoT profile in Certificate Manager
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
+	-   Fixed a issue of target directory not being able to set after invalid path.
```

Comment:
Fixed an issue of target directory, now it is possible to set the target directory after an invalid path.

If we rephrase, the technical accuracy is intact?